### PR TITLE
fix: display swap button in top assets only on hover

### PR DESCRIPTION
**Pull request by compojoom**
## What it solves
Display the swap button only on hover in the "top asset" block. 
@TanyaEfremova @kirkkonen looks a bit weird or like a bug, because we have this space on the right now without the button.

## Screenshots
grafik
grafik

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/components/dashboard/PendingTxs/styles.module.css [hunk 1]
```diff
@@ -11,11 +11,19 @@
   min-height: 50px;
 }
 
+.container button {
+  opacity: 0;
```
Comment (katspaugh) [inline src/components/dashboard/PendingTxs/styles.module.css:15]:
```suggestion
  opacity: 0;
  transition: opacity .1s ease-in;
```

**Comment by katspaugh** [comment]
I would keep it w/o hover tbh.

**Review by katspaugh — APPROVED** [review]

**Comment by francovenica** [comment]
I think the change is fine, but for mobile viewport, since there is no "hover", can we keep it always showing?

Note: technically if you keep pressing the asset row the button shows up, but with no indicator that that's there, nobody will do that.

image

**Comment by katspaugh** [comment]
I'll go ahead and close this PR because:
* people do click on these buttons (second most popular entry point to Swaps after the sidebar)
* the layout become unbalanced until you hover due to the empty space on the right

**Pull request closed without merging**